src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-storage-class-form/rgw-storage-class-form.component.html

```diff
@@ -324,18 +284,18 @@
               [invalidText]="readThroughError"
             ></cds-number>
             <ng-template #readThroughError>
-              <span
-                class="invalid-feedback"
-                *ngIf="storageClassForm.showError('read_through_restore_days', formDir, 'pattern')"
-                i18n
-                >The entered value must be a positive integer.</span
-              >
-              <span
-                class="invalid-feedback"
-                *ngIf="storageClassForm.showError('read_through_restore_days', formDir, 'lockDays')"
-                i18n
-                >ReadThrough Restore Days must be positive.</span
-              >
+              @if (storageClassForm.showError('read_through_restore_days', formDir, 'pattern')) {
+              <span class="invalid-feedback"
+                    i18n>
+                The entered value must be a positive integer.
+              </span>
+              } @else if (storageClassForm.showError('read_through_restore_days', formDir,
+              'lockDays')) {
+              <span class="invalid-feedback"
+                    i18n>
+                ReadThrough Restore Days must be positive.
+              </span>
```

Comment:
both errors seem same, can we better sentence the later or check if we need both ?
Or just explain me if I am wrong here :)